src/Makefile.gtest.include

```diff
@@ -86,12 +88,13 @@ pastel_gtest_SOURCES +=\
 	
 if ENABLE_WALLET
 pastel_gtest_SOURCES += \
+	wallet/gtest/test_accounting.cpp\
 	wallet/gtest/test_wallet.cpp\
 	wallet/gtest/test_wallet_ismine.cpp
 endif
 
 
-pastel_gtest_CPPFLAGS = $(AM_CPPFLAGS) -DBINARY_OUTPUT -DCURVE_ALT_BN128 -DSTATIC $(BITCOIN_INCLUDES) -I$(srcdir)/gtest
+pastel_gtest_CPPFLAGS = $(AM_CPPFLAGS) -DBINARY_OUTPUT -DCURVE_ALT_BN128 -DSTATIC $(BITCOIN_INCLUDES) -I$(srcdir)/gtest -I$(builddir)/test/
```

Comment:
remove that include, is idea is to remove any references to bitcoin_test

Reply:
Fixed. I removed that include and add the raw data generation in Makefile.gtest.include